templates/createDropbox.hbs

```diff
@@ -50,38 +50,52 @@
   </form>
 </section>
 {{#if dropbox.hasUploads }}
-  <section>
-    <h2>2. Add your details:</h2>
-    <p>
-      Please fill in as much information as possible so we can identify you. When you are done, click "Submit Documents"
-      to send the documents to us.
-    </p>
-    <form action="" method="POST">
-      <div class="form-group">
-        <label for="customerName">Your name</label>
-        <input type="text" class="form-control" id="customerName" name="customerName" value="{{ dropbox.customer.name }}"
-          required />
-      </div>
-      <div class="form-group">
-        <label for="customerEmail">Email</label>
-        <input type="text" class="form-control" id="customerEmail" name="customerEmail"
-          value="{{ dropbox.customer.email }}" />
-      </div>
-      <div class="form-group">
-        <label for="customerPhone">Phone number</label>
-        <input type="text" class="form-control" id="customerPhone" name="customerPhone"
-          value="{{ dropbox.customer.phone }}" />
-      </div>
-      <div class="form-group">
-        <label for="description">Why are you uploading these documents?</label>
-        <textarea class="form-control" id="description" name="description" rows="3"
-          placeholder='Give a reason for uploading the documents, eg. "to apply for change of circumstance"'
-          required>{{ dropbox.description }}</textarea>
-      </div>
-      <div class="form-group">
-        <button type="submit" id="submitDropbox" class="btn btn-block btn-lg btn-primary">Submit Documents</button>
-      </div>
-    </form>
-  </section>
+<section>
+  <h2>2. Add your details:</h2>
+  <p>
+    Please fill in as much information as possible so we can identify you. When you are done, click "Submit Documents"
+    to send the documents to us.
+  </p>
+  <form action="" method="POST">
+    <div class="form-group">
+      <label for="customerName">Your name</label>
+      <input type="text" class="form-control" id="customerName" name="customerName" value="{{ dropbox.customer.name }}"
+        required />
+    </div>
+    <div class="form-group">
+      <label for="customerPhone">Phone number</label>
+      <input type="text" class="form-control" id="customerPhone" name="customerPhone"
+        value="{{ dropbox.customer.phone }}" />
+    </div>
+    <div class="form-group">
+      <label for="customerDob">Date of birth:</label>
+      <input type="date" class="form-control" id="customerDob" name="customerDob" value="{{ dropbox.customer.dob }}" />
+    </div>
+    <div class="form-group">
+      <label for="customerEmail">Email</label>
+      <input type="text" class="form-control" id="customerEmail" name="customerEmail"
+        value="{{ dropbox.customer.email }}" />
+    </div>
+    <div class="form-group">
+      <label for="customerPhone">Phone number</label>
+      <input type="text" class="form-control" id="customerPhone" name="customerPhone"
+        value="{{ dropbox.customer.phone }}" />
+    </div>
```

Comment:
We have phone number twice? (also at L65)

Reply:
Oh dear! Thank you !